src/shared/layout/header/header.scss

```diff
@@ -20,3 +20,53 @@ $header-color-hover: darken($header-color, 20%);
   top: 0px;
   z-index: 1031;
 }
+
+@media (min-width: 64em) {
+  .flex-container {
+    display: flex;
+    flex-wrap: nowrap;
+  }
+  .banner {
+    padding-left: 0rem;
+  }
+  .hud-logo {
+    width: 140px;
+    height: 130px;
+    // margin-bottom: 0.9rem;
+  }
+  .header-txt {
+    color:white;
+    line-height: 50px;
+    font-size: 2.9rem;
+    font-weight: 600;
+    margin: 2rem 0 1.5rem 1rem;
+  }
+  .txt-container {
+    margin-top: 2.5rem;
+    margin-left: 2.5rem;
+  }
```

Comment:
It appears the banner is wider than it used to be.  There seems to be a lot more blue to the right of the banner than in the samples the customer provided.  Because it is wider, it seems to dwarf the form even more.  What do you think?

![image](https://user-images.githubusercontent.com/70591280/107250781-ff8eeb00-6a01-11eb-98d4-7b75503c8127.png)